simulation.py

```diff
@@ -46,14 +47,18 @@ def run_simulation(request_log_file, enable_concurrency = True, request_map = No
 
   args = vars(parser.parse_args())
 
+  #greedy = GreedyReplication()
+  #greedy.last_timestamp = int(time.time())
+
   print '************************* Running simulation *************************'
-  before_volley, start_time, end_time, request_map = run_simulation('dataset/sample_log_ready', args['disable_concurrency'])
-  Volley(start_time, end_time).execute()
-  after_volley, start_time, end_time, request_map = run_simulation('dataset/sample_log_ready', False, request_map)
-  # greedy = GreedyReplication()
+  before_volley, start_time, end_time, request_map, uuid_to_server = run_simulation('dataset/sample_log_ready', args['disable_concurrency'])
```

Comment:
Hmmm. After merging in we should go back to your pull request about executing volley/greedy/distributed using flags.. Sorry for not looking at it earlier!